ci/test_integtation

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash -ex
+
+#
+# Local Integration Test Runner
+#
+# This script will run all features marked with
+# `not @integration`, which avoids the usage of a live
+# remote PCF instance, or any needed credentials.
+#
+
+function finish {
+  echo 'Removing environment'
+  echo '-----'
+  docker-compose down -v
+}
+trap finish EXIT
+
+cd "$(dirname $0)"
+
+# Set up the containers to run in their own namespace
+COMPOSE_PROJECT_NAME="$(basename "$PWD")_$(openssl rand -hex 3)"
+export COMPOSE_PROJECT_NAME
+
+export BRANCH_NAME=${BRANCH_NAME:-$(git symbolic-ref --short HEAD)}
+
+# Sets up conjur and retrieves credentials
+. ./start_conjur
+
+# Run the install script to verify compatibility
+docker-compose run --rm tester bash ./lib/install_go.sh
```

Comment:
Perhaps unrelated to the core of the PR...

Noticed that we're doing this all over the place
```
docker-compose run --rm tester bash ./lib/install_go.sh
```

What seems like being verbose by prepending `bash`. I'm not sure why. I would have thought without bash would suffice...

```
docker-compose run --rm tester ./lib/install_go.sh
```

Reply:
Yeah that works, I'll strike the `bash` command in there, thanks!